packages/schema-dot-org-json-ld-components/README.md

```diff
@@ -0,0 +1,19 @@
+[Schema.org](http://schema.org) React component – schema-dot-org-json-ld-components
+====================================================================================
+This is an unnoficial attempt to facilitate rendering the json-ld format of schema.org
+markup using [React](https://facebook.github.io/react/).
+
+Usage
+-----
+```
+npm install -S schema-dot-org-types
+npm install -S schema-dot-org-json-ld-markup
+```
+
+```
+//@flow
+import {VideoObject} from 'schema-dot-org-types';
+ipmort type {VideoObjectType} from 'schema-dot-org-types/lib/types';
```

Comment:
I'll get there. Not sure I can do it before I have to publish this for use in home-details-content.